example/tmux.yaml

```diff
@@ -0,0 +1,203 @@
+---
+  #This config will not currently function and will error out until Issue #248 is implemented, or another solution is.
```

Comment:
Sorry, maybe I didn't articulate myself well enough. I really liked the file before but didn't want to confuse users without non implemented features yet. That's why I proposed to split them, the overview is awesome now. This file shouldn't error out. This should be a as good as we can get to parity at the moment. But you can rename this file and keep it too, so we have a good start if the configurable modes are implemented, if you want.

Reply:
oh oops, edited the wrong yaml, that is now fixed. 👍 

I've renamed that file to a .nonfunctional for now, so its available, but isn't shown as a yaml.

Reply:
Thanks!